src/async_pools/mongoose_aggregator_worker.erl

```diff
@@ -158,9 +158,9 @@ terminate(Reason, State) ->
 code_change(_OldVsn, State, _Extra) ->
     {ok, State}.
 
-% Don't leak the tasks to logs, can contain private information
```

Comment:
Small issue, but this comment was left on L114 in `mongoose_batch_worker`. I think it was supposed to be left here as well?